Customizable container div

Zucchy00 opened the pull request:
# Implemented the feature to allow the user to change the container class as he prefer.

### As this could not be possibile without it:

### Current:
```
<Tooltip tip="This is a button" bottom>
    <button class="h-[90%] aspect-square rounded-full ring ring-white hover:scale-[110%] hover:bg-neutral-500/30 hover:cursor-pointer transition-all duration-200 flex items-center justify-center">
        <div class="aspect-square h-[90%]">
            <Plus class="w-full h-full"/>
        </div>
    </button>
</Tooltip>
```
### Modified:
```
<Tooltip tip="This is a button" container="h-[90%] aspect-square" bottom>
    <button class="w-full h-full rounded-full ring ring-white hover:scale-[110%] hover:bg-neutral-500/30 hover:cursor-pointer transition-all duration-200 flex items-center justify-center">
        <div class="aspect-square h-[90%]">
            <Plus class="w-full h-full"/>
        </div>
    </button>
</Tooltip>
```

Zucchy00 committed:
Implemented the feature to allow the user to change the container class as he prefer

sharu725 commented:
Nice. I'm still checking this. In the meantime, can you update to the code from
```svelte
<span class={`tooltip-slot ${container}`} use:tooltip>
```

to this?
```svelte
<span class="tooltip-slot {container}" use:tooltip>
```


Also, a better way to do it would be to use ``class`` instead of ``container``

```svelte
  let className;
  export { className as class };

<span class="tooltip-slot {className}" use:tooltip></span>
```

Now, it can be used like this

```svelte
<Tooltip class="h-20"></Tooltip>
```

But the real issue is that I have some predefined styles for the ``Tooltip`` element already. I will have to allow overwrite with other styles. I'm not quite sure how at this point. 
Let me think about it.

Zucchy00 committed:
implemented class and style to the div container

Zucchy00 commented:
Changed as you prefered, I think you could set a parameter as boolean called "overwrite" to change the styles of the tooltip. Also added a style parameter to improve the customization
